src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/UnusedLocalVariableCheckTest.java

```diff
@@ -205,8 +205,8 @@ public void testUnusedLocalVarNestedClasses() throws Exception {
             "24:13: " + getCheckMessage(MSG_UNUSED_LOCAL_VARIABLE, "Q"),
             "36:9: " + getCheckMessage(MSG_UNUSED_LOCAL_VARIABLE, "variable"),
             "44:9: " + getCheckMessage(MSG_UNUSED_LOCAL_VARIABLE, "anotherVariable"),
-            "67:21: " + getCheckMessage(MSG_UNUSED_LOCAL_VARIABLE, "s"),
-            "68:21: " + getCheckMessage(MSG_UNUSED_LOCAL_VARIABLE, "n"),
+            "71:21: " + getCheckMessage(MSG_UNUSED_LOCAL_VARIABLE, "s"),
+            "72:21: " + getCheckMessage(MSG_UNUSED_LOCAL_VARIABLE, "n"),
```

Comment:
Please create completely new test case with exact config from issue instead of extending existing tests. Make sure to add previous false positives found in reports to test cases.